packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -359,10 +361,16 @@ function transform(root: IRNode, codeGen: CodeGen): t.Expression {
         templateContainsId: boolean
     ): t.Expression {
         const { namespaceURI, tagName } = element.__original as parse5.AST.Default.Element;
+        const isUsedAsAttribute = isAttribute(element, attr.name);
```

Comment:
I don't think this is needed since the `computeAttrValue` only runs for attributes.

Reply:
this was my first impression, but i was wrong, it's used to generate the code for both [props](https://github.com/salesforce/lwc/blob/master/packages/@lwc/template-compiler/src/codegen/index.ts#L467-L472) and [attrs](https://github.com/salesforce/lwc/blob/master/packages/@lwc/template-compiler/src/codegen/index.ts#L459-L464)